CMakeLists.txt

```diff
@@ -185,6 +191,7 @@ endif()
 if(WITH_MKL)
   find_path(MKL_ROOT include/mkl.h DOC "Path to MKL root directory" PATHS
     $ENV{MKLROOT}
+    ${INTEL_ROOT}/mkl/latest
```

Comment:
On which platform is this path valid?

Reply:
This is on Windows. They changed the directory structure in oneAPI.